src/query/xasl.h

```diff
@@ -1034,6 +1034,14 @@ struct xasl_node
   int cte_host_var_count;
   int *cte_host_var_index;
 
+  MHT_TABLE *sq_cache_ht;
+  int sq_cache_flag;
+  size_t sq_cache_hit;
+  size_t sq_cache_miss;
+  size_t sq_cache_size;
+  size_t sq_cache_size_max;
+  size_t sq_cache_miss_max;
```

Comment:
It is not good for XASL_NODE to have statistical information directly.
Please see the following:
```
struct xasl_node
{
  ...
#if defined (SERVER_MODE) || defined (SA_MODE)
  ORDERBY_STATS orderby_stats;
  GROUPBY_STATS groupby_stats;
  XASL_STATS xasl_stats;
  ...
#endif
  ...
};
```